quasar.conf.js

```diff
@@ -0,0 +1,276 @@
+/*
+ * This file runs in a Node context (it's NOT transpiled by Babel), so use only
+ * the ES6 features that are supported by your Node version. https://node.green/
+ */
+
+// Configuration for your app
+// https://quasar.dev/quasar-cli/quasar-conf-js
+/* eslint-env node */
+
+const { configure } = require('quasar/wrappers')
+const { resolve } = require('path')
+const fs = require('fs')
+const StyleLintPlugin = require('stylelint-webpack-plugin')
+const PreloadWebpackPlugin = require('preload-webpack-plugin')
+const HardSourceWebpackPlugin = require('hard-source-webpack-plugin')
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
+
+function getHttpsOptions () {
+  /* Try to set up https with your own cert for usage with mkcert
+  Define your cert path and filenames in a file called .env (this uses dotenv)
+
+  KEY=key.pem
+  CERT=cert.pem
+  CA=/home/tic/.local/share/mkcert/rootCA.pem
+
+  More info: https://github.com/FiloSottile/mkcert
+  If these are not available, fall back to making our own cert
+  */
+  try {
+    return {
+      key: fs.readFileSync(process.env.KEY),
+      cert: fs.readFileSync(process.env.CERT),
+      ca: fs.readFileSync(process.env.CA),
+    }
+  }
+  catch (e) {
+    console.log('Could not find key/cert/ca files, falling back to our own...', e)
+    return true
```

Comment:
This stuff causes this to printed out on `yarn build`:

```
Could not find key/cert/ca files, falling back to our own... [  Error: TypeError [ERR_INVALID_ARG_TYPE] [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type   string or an instance of Buffer or URL. Received undefined
```

Maybe check if env vars are set...